modules/virt-configuring-secondary-dns-server.adoc

```diff
@@ -38,14 +63,14 @@ NAME       TYPE             CLUSTER-IP     EXTERNAL-IP      PORT(S)          AGE
 dns-lb     LoadBalancer     172.30.27.5    10.46.41.94      53:31829/TCP     5s
 ----
 
-. Edit the `HyperConverged` CR in your default editor by running the following command:
+. Open the `HyperConverged` CR configuration again:
```

Comment:
1. Why replacing "edit" with "open"? The user should be instructed to edit the CR, just like he is instructed to do so above when enabling `deployKubeSecondaryDNS`.
2. "configuration" seems redundant. The `HyperConverged` resource is a CR.

Reply:
Good points, will update, thank you.